test/TestConfig/scripts/build_test.xml

```diff
@@ -95,38 +87,4 @@
 
 		<chmod file="${ascii.test.dir}/*.mk" perm="755"/>
 	</target>
-
-
-	<target name="check_sdk">
```

Comment:
We should keep `check_sdk ` for now as it is still in use.

Reply:
check_sdk has been reserved for the uses of other repos.